lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -68,7 +68,14 @@ export default class ExistingEksOpenSourceobservabilityPattern {
                 __dirname + '/../common/resources/amp-config/java/alerting-rules.yml',
```

Comment:
We are missing a boundary condition here here both Java and API Server pattern is enabled which means it uses open telemetry operator from common folder so we are still missing the annotation to select.